pylint_odoo/checkers/modules_odoo.py

```diff
@@ -829,19 +829,17 @@ def _get_module_files(self):
 
     def _check_file_not_used(self):
         """Check if a file is not used from manifest"""
-        self.msg_args = []
         module_files = set(self._get_module_files())
         referenced_files = set(self._get_manifest_referenced_files()).union(
             set(self._get_xml_referenced_files())
         )
-        for no_referenced_file in (module_files - referenced_files):
-            if (not no_referenced_file.startswith('static/') and
-                not (no_referenced_file.startswith('test/') or
-                     no_referenced_file.startswith('tests/'))):
-                self.msg_args.append((no_referenced_file,))
-        if self.msg_args:
-            return False
-        return True
+        excluded_dirs = ['static', 'test', 'tests', 'migrations']
+        no_referenced_files = [
+            file for file in (module_files - referenced_files)
+            if file.split('/')[0] not in excluded_dirs
```

Comment:
Use os.path methods

Reply:
Done, used as much as I could. Since there's no way to extract the top directory from a path using `os.path`, I used `os.path.sep`